docs/stories/21 - Deduction.adoc

```diff
@@ -0,0 +1,128 @@
+
+The groan, however, wasn't for his aching muscles or the uncertain future. It was for the venerable gentleman approaching him in the twilight.
+
+"Kitsuki-san," Hitori offered in greeting, offering a genuinely respectful bow. He might not like Masanori but the old justicar was due a great deal of respect. For his part, Masanori offered a proper bow in turn.
+
+"Mirumoto-sama. My congratulations on a successful journey." He fished out a pouch of smoke leaf and began the process of loading his pipe. "It reminds me of a time I'd traveled down south. Earthquake Fish Bay, if you must know. It was a simple thing, a merchant patron of the Yasuki - one who'd maintained family connections to cousins (by marriage) in the Crane - had been under pressure from her wife to enrich their personal accommodations. By sheer coincidence a number of funds had gone missing, including from a certain shipping combine
+with lumber ties to another certain province in the lands of the Dragonfly where I-"
+
+"Kitsuki-san, you know how much I enjoy the tales of your career and the myriad criminals you have brought to justice. But if you have a point to make, please come to it. It's very late and we are all very tired."
+
+Masanori smiled as he tapped his pipe. "You remind me of my own wife, my lord, gone three years now, bless her memory. Very well. If we're to stay here, we need to explore these lands. To that end, my young cousin," he glossed over how distant, "Seiji is perhaps singularly trained to understand them. Before he goes out, I want his safety to be seen to. Ergo, I would ask you to assign Mirumoto Rei as his yojimbo."
+
+Hitori considered the matter. Rei was the most senior of the  rank-and-file bushi that had accompanied them. He could certainly spare her for this task, and she'd likely be quite good at it, but odd that Masanori had requested Rei specifically. He could stall and look into the matter. By the gods, with his authority he could even ask Masanori outright why he wanted Rei as Seiji's yojimbo, though that was likely a poor tactic. They were going to be working very closely for the rest of their lives in these lands; no need to start things off with such antagonism. Better to get a favor.
+
+"I'll speak to her first thing in the morning, though what you think to find with one of Seiji's talents in these places is beyond..."
+
+His voice trailed off as a bright blue glow appeared on his face, reflecting the lights now shimmering in the waters below.
+
+"You know," said Hitori as he found his voice. "I think Seiji and Rei would work well together. Tell them both tomorrow there is an assignment for them."
+
+'''
+
+{empty} +
```

Comment:
same here, the `empty` clauses are most likely not required.

Reply:
But you should add a `<<<`, so that the page break is enforced. Otherwise, one single sentence is at the end of the page.